cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,3 +1,3 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false --log-level=warning"
```

Comment:
Both work, judging by the code:
https://github.com/docker/docker/blob/729c9a97822ebee2c978a322d37060454af6bc66/vendor/src/github.com/Sirupsen/logrus/logrus.go#L43